tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md

```diff
@@ -92,15 +103,43 @@ Cloud Platform (GCP).
         spring.datasource.username=root
         spring.datasource.password=my-smart-password
 
+    See updated file [here](spring-petclinic/src/main/resources/application-mysql-datasource.properties).
+
 1.  Update `pom.xml` to include [Cloud SQL MySQL Socket Factory][socket].
     The socket library allows you to connect to your Cloud SQL instance for
-    local testing and deployment.
+    local testing and deployment. See reference `pom.xml` [here](spring-petclinic/pom-spring-datasource.xml).
```

Comment:
Change to: "Refer to [`pom-spring-datasource.xml`](spring-petclinic/pom-spring-datasource.xml)."